docs/modules/ROOT/pages/how-tos/custom-theme.adoc

```diff
@@ -40,25 +40,33 @@ parameters:
   keycloak:
     extraInitContainers:
       theme-provider:
-        image: company/keycloak-theme:v1.0.0
+        image: company/keycloak-themes:v1.0.0
         imagePullPolicy: IfNotPresent
         command:
           - sh
         args:
           - -c
           - |
             echo "Copying theme..."
-            cp -R /theme/* /company-theme
+            cp -R /themes/* /target/
         volumeMounts:
-          - name: theme
-            mountPath: /company-theme
+          - name: themes
+            mountPath: /target
+
     extraVolumes:
       theme:
         emptyDir: {}
+
     extraVolumeMounts:
-      theme:
-        readOnly: true
+      - name: theme
         mountPath: /opt/jboss/keycloak/themes/company
+        subPath: company
+        readOnly: true
+      - name: theme
+        readOnly: true
+        mountPath: /opt/jboss/keycloak/themes/other-theme
+        subPath: other-theme
+        readOnly: true
```

Comment:
Is this page up-to-date? Somehow the volumeMount `themes` doesn't match `theme` in the `extraVolumeMounts` array?

Reply:
Updated the docs to be more consistent. I used multiple themes by default since it's easier to deduct how a single theme works from there